Deque/deque_test.py

```diff
@@ -1,54 +1,40 @@
 import unittest
 
 from Deque.deque import Deque
-from collections import deque
 
 
 class TestQuickSort(unittest.TestCase):
 
     def test_push_front(self):
         my_dq = Deque()
```

Comment:
It's better to name those variables "actual_value" for values your function returns, and "expected_value" for expected ones.

Reply:
renamed